src/smart-components/workspaces/WorkspaceListTable.tsx

```diff
@@ -1,17 +1,24 @@
-import React, { Suspense, useEffect, useMemo } from 'react';
+import React, { Suspense, useEffect, useMemo, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { Outlet, useSearchParams } from 'react-router-dom';
-import { useIntl } from 'react-intl';
+import { Outlet } from 'react-router-dom';
+import { FormattedMessage, useIntl } from 'react-intl';
 import { fetchWorkspaces } from '../../redux/actions/workspaces-actions';
-import { BulkSelect, BulkSelectValue, ErrorState, ResponsiveAction, ResponsiveActions, SkeletonTableBody } from '@patternfly/react-component-groups';
+import {
+  BulkSelect,
+  BulkSelectValue,
+  ErrorState,
+  ResponsiveAction,
+  ResponsiveActions,
+  SkeletonTableBody,
+  SkeletonTableHead,
+} from '@patternfly/react-component-groups';
 import {
   DataView,
+  DataViewState,
```

Comment:
We still need the `DataViewTextFilter` and `useDataViewFilters`
```suggestion
  DataViewState,
  DataViewTextFilter,
  useDataViewFilters,
```